### PR TITLE
Add practice exercise `wordy`

### PR DESCRIPTION
**Pull request by natanaelsirqueira**
Wanted to have some fun so picked a more complex one. :)

### CODE REVIEW

File: config.json [hunk 1]
```diff
@@ -1178,6 +1178,25 @@
...
+          "result",
+          "string",
+          "case",
+          "list",
+          "bool",
+          "int",
+          "maths",
+          "custom-type"
+        ],
+        "difficulty": 8
```
Comment (natanaelsirqueira) [inline config.json:1198]: This exercise has difficulty 7 in Elixir. I gave it one level more here because it's a bit tricky to handle the different edge cases and you must be comfortable with the type system and a few different concepts.

File: exercises/practice/wordy/test/wordy_test.gleam [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+  wordy.answer("What is 1 2 plus?")
+  |> should.equal(Error(SyntaxError))
+}
+
+pub fn reject_prefix_notation_test() {
+  wordy.answer("What is plus 1 2?")
+  |> should.equal(Error(SyntaxError))
+}
+
+pub fn reject_division_by_zero_test() {
```
Comment (natanaelsirqueira) [inline exercises/practice/wordy/test/wordy_test.gleam:124]: I had to adapt the solution a bit to normalize the operations since `int.divide` is the only one that returns a `Result`, so I figured I'd add this extra test case for the sake of it.

File: exercises/practice/wordy/.meta/example.gleam [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+
+fn evaluate(tokens: List(Token)) -> Result(Int, Error) {
+  try [Operand(result)] =
+    list.try_fold(
+      tokens,
+      [],
+      fn(previous_tokens: List(Token), current_token: Token) {
+        case previous_tokens {
+          [Operator(operation), Operand(left_operand)] -> {
+            assert Operand(right_operand) = current_token
```
Comment (lpil) [inline exercises/practice/wordy/.meta/example.gleam:55]: Could be nice to return an error here rather than crashing, especially as this function already is a `Result` returning function.

**Review by lpil — APPROVED** [review]
Oh this is very nice! I left one request and a small suggestion in the example. Great stuff!

File: exercises/practice/wordy/.meta/example.gleam (hunk 3 above)
Comment (natanaelsirqueira) [reply exercises/practice/wordy/.meta/example.gleam]: Nice. I did this because the sequence of tokens was already being validated at the tokenization phase, so I thought it would be redundant to check for an error here. Now I realize that it makes more sense to do it here and leave the `parse_tokens` function cleaner since this validation is only relevant to this particular implementation of the `evaluate` function.

I like this solution better now. :)

**Review by lpil — APPROVED** [review]
Oh I **really** like this new version, fantastic!
